packages/ember/addon/instance-initializers/sentry-performance.ts

```diff
@@ -22,7 +22,7 @@ export function initialize(appInstance: ApplicationInstance): void {
 
 function getTransitionInformation(transition: any, router: any) {
   const fromRoute = transition?.from?.name;
-  const toRoute = transition ? transition.to.name : router.currentRouteName;
+  const toRoute = transition && transition.to ? transition.to.name : router.currentRouteName;
```

Comment:
I don't know how it can happen but, in my EmberJS project using `ember-simple-auth` after a login the `transition` object from the login page to the default authenticated page contains no `to` key.
`route.currentRouteName` is well defined.

This defensive programming contribution fixes my issue.